Web/starter_project_iconics/pages/blog/add-blog.tsx

```diff
@@ -41,20 +59,69 @@ const AddBlog: React.FC = () => {
           <TextEditor />
         </div>
         <div className="flex justify-end gap-x-7 font-montserrat font-semibold mb-8 mt-8">
-          <a href="#" className="pt-3 text-blue-600">
+          <Link href="#" className="pt-3 text-blue-600">
             Cancel
-          </a>
+          </Link>
           <button className="text-white bg-blue-600 py-2 px-4 border-gray-400 rounded shadow">
             Save Changes
           </button>
         </div>
       </div>
 
       <div
-        className="col-span-1 border-l-2 border-indigo-100 mt-8"
+        className="col-span-3 border-l-2 border-indigo-100 m-8 sm:col-span-1 sm:width-1/2"
         style={{ height: '575px' }}
       >
-        <h3 className="font-bold ml-4">Select Tag</h3>
+        {isLoading ? (
```

Comment:
can you handle the isLoading part by itself instead of inside the HTML code? look at the impact-stories page in the home folder for reference.